cmake/partition_manager.cmake

```diff
@@ -522,4 +522,22 @@ else()
   set(ZEPHYR_RUNNER_CONFIG_KERNEL_HEX "${final_merged}"
     CACHE STRING "Path to merged image in Intel Hex format" FORCE)
 
+  # Zephyr has a Kconfig option used for signing an application image
+  # with MCUboot using west sign. If partition manager is in use and
+  # there are multiple images, we want to make sure users understand
+  # this option should probably be left alone, since the NCS build
+  # system has its own way of managing signing.
+  get_property(NCS_WARNED_ON_MCUBOOT_SIGNATURE_KEY_FILE GLOBAL PROPERTY
+    NCS_WARNED_ON_MCUBOOT_SIGNATURE_KEY_FILE SET)
```

Comment:
What is the purpose of the `NCS_WARNED_ON_MCUBOOT_SIGNATURE_KEY_FILE ` property ?
I see you read it here, and you set it in line 541, but I don't see you use it for anything.

Reply:
> What is the purpose of the `NCS_WARNED_ON_MCUBOOT_SIGNATURE_KEY_FILE ` property ?
> I see you read it here, and you set it in line 541, but I don't see you use it for anything.

I don't really understand the logic in this file and I want to make sure I only warn once.

Reply:
... and I got that wrong ;)